tests/test_constants.py

```diff
@@ -18,3 +24,19 @@ def test_supported_file_types():
 def test_supported_databases():
     expected = ["bigquery", "postgres", "snowflake", "sqlite"]
     assert sorted(SUPPORTED_DATABASES) == expected
+
+
+@mock.patch.dict(
+    os.environ, {"AIRFLOW__CORE__ENABLE_XCOM_PICKLING": "False"}, clear=True
+)
+def test_enable_xcom_pickling_set_false_in_env():
+    import astro
```

Comment:
Perhaps this could be on top of the file, WDYT?

Reply:
The reason we need that import was just for the tests. That was the reason I kept it inside tests